Vesper-Agents-Suite/Agent7/src/agent.spec.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  TransactionEvent
+} from "forta-agent"
+import { TestTransactionEvent, encodeParameters} from "forta-agent-tools";
+import agent from "./agent"
+
+import {
+  NEW_IMPLEMENTATION_SIGNATURE,
+  COMPOUND_COMPTROLLER_ADDRESS,
+  NEW_IMPLEMENTATION_EVENT_ALERT_ID
+} from "./utils"
+
+describe("high gas agent", () => {
+  let handleTransaction: HandleTransaction
+  const mockAddress = '0xB44ddb00B19a2F20E3c1C31A14C8965D75a8e4De'
+  const oldAddress1 = '0x7042499A36c4c572bf9C1e9cD384F63FaF10800A'
+  const newAddress1 = '0x97AF9877E4a815b532153422F5B4127F296a1487'
+  const oldAddress2 = '0x49f6207C0cB311cA52f2AA0D73825bC75D57fB1e'
+  const newAddress2 = '0x79a08F3Bf26Ad269a0832bd3467F236c9D0257d6'
+
+  beforeAll(() => {
+    handleTransaction = agent.handleTransaction
+  })
+
+  describe("handleTransaction", () => {
```

Comment:
If you only have the `handleTransaction` you may not need to add this level and you can go with a more flattened structure.